molior/api/status.py

```diff
@@ -131,4 +131,15 @@
     backend = b.get_backend()
     build_nodes = backend.get_nodes_info()
     # uptime_string = str(timedelta(seconds = uptime_seconds))
-    return web.json_response(build_nodes)
+    results = []
+    for name in build_nodes:
+        load = ""
+        for l in build_nodes[name]["load"]:
```

Comment:
", ".join(build_nodes[name]["load"])?

Reply:
the load array contains floats, join does not work on floats ;(